test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -480,23 +480,24 @@ public static MockServer CreateMockServer(IList<IPackage> packages)
         /// Path to nuget.exe for tests.
         /// </summary>
         public static string GetNuGetExePath()
-        {
-            return _nuGetExePath.Value;
-        }
-
-        private static readonly Lazy<string> _nuGetExePath = new Lazy<string>(GetNuGetExePathCore);
-
-        private static string GetNuGetExePathCore()
         {
             var targetDir = ConfigurationManager.AppSettings["TestTargetDir"] ?? Directory.GetCurrentDirectory();
-            var nugetexe = Path.Combine(targetDir, "NuGet", "NuGet.exe");
+            var fileName = "NuGet.exe";
```

Comment:
If it's constant then make it actual constant and move to top of file?

Reply:
Isn't making it constant within the method enough? It is a static method, though...?